back-end/java/main/java/src/DormRoom/CampusLocation.java

```diff
@@ -0,0 +1,24 @@
+package src.DormRoom;
+
+/**
+ * An enum representing general campus areas where a dorm room
+ * building can be located.
+ */
+public enum CampusLocation {
+  /**
+   * East Campus area.
```

Comment:
Similar to bathroom type, we WILL have more location types so make the code easy to change